frontend/src/lib/i18n/locales/en.json

```diff
@@ -182,6 +182,15 @@ the [Linguistics Institute at Payap University](https://li.payap.ac.th/) in Chia
       "user_not_verified": "User has not verified their email address",
       "user_needs_to_relogin": "Added members will need to log out and back in again before they see the new project.",
     },
+    "bulk_add_members": {
```

Comment:
I think we need to make it clear that this feature **creates** users. It's actually very different to the "Add Member" feature.
How about this:
```json
"bulk_add_members": {
  "add_button": "Bulk Add/Create Members",
   "explanation": "Adds all the provided Send/Receive logins to this project. New users will be created automatically with the provided password for logins that don't exist.", // I'd put this at the top under the modal title
  "modal_title": "Add/Create Members",
  "submit_button": "Add/Create Members",
  "shared_password": "Shared Password",
  "usernames": "Send/Receive Logins (one per line)", // <-- "login" is the term used in FLEx and (previously) Redmine, so it's the term people are familiar with and the one we use on the login page
  "project_not_found": "Project not found. Please refresh the page.", // <-- I know we have an error for this on the Add member mutation/workflow, but as far as I can tell, we shouldn't need it
},
```

Reply:
I really don't like the term "logins" when "usernames" is standard, and the target audience is admins, not the general public. I suppose "logins/usernames" is sufficient.

Reply:
Done in commit 848d045f311f8358661b7b8770cf357ee53c58ea.

Reply:
My comment was a bit confusing, because it was actually a compilation of a lot of rewording suggestions (or it looks like you may have muddled up my button text suggestions):
- `"add_button": "Bulk Add/Create Members"` I think the external button is the (only) one that really needs to be explicit about what this feature is capable of.
- `"modal_title": "Add/Create Members"` I find your current title quite wordy. I think this is sufficient. Adding "bulk" here would be fine to me, but it's not necessary. It's similar to:
- `"submit_button": "Add/Create Members"`, I think the word "bulk" is superfluous here. It doesn't hurt, but it's very obvious at this point that it's a bulk tool.

Reply:
@rmunn can you reply to my last comment here? 🙂

Reply:
Yeah, I missed the `add_button` changes earlier; thanks for explicitly mentioning them. Done in commit b218ff8d59e03f7fcf1f0c16bad27be1f1b5c0db where I've also gone ahead and made the other changes you mentioned in https://github.com/sillsdev/languageforge-lexbox/pull/621#discussion_r1514714186.